Calculation.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculatorLibrary
```

Comment:
C# these days supports stand alone name spaces. All you need is to add a semi-colon. Again it improves readability: 
![image](https://user-images.githubusercontent.com/108711476/202833555-64427440-a611-4ad4-b812-26ad17a67921.png)